data/na/swagger/1-0-5.yaml

```diff
@@ -786,6 +786,34 @@ paths:
           $ref: '#/responses/MethodNotAllowed'
         500:
           $ref: '#/responses/InternalServerError'
+  /recurringPayment:
+  '/recurring/{accountId}':
+    get:
+      summary: Get account
+      description: 'Get a Recurring Payment Profile using account ID, also known as ID.'
```

Comment:
is it known as ID or Billing ID (from UI)?

Reply:
it is ID